README.md

```diff
@@ -234,18 +251,19 @@ Use the HTTP proxy to the connect the API endpoints.
 
 We’d love to hear your thoughts on this project. Feel free to drop us a note.
 
-* [Twitter](https://twitter.com/charmcli)
-* [The Fediverse](https://mastodon.social/@charmcli)
-* [Discord](https://charm.sh/chat)
+- [Twitter](https://twitter.com/charmcli)
+- [The Fediverse](https://mastodon.social/@charmcli)
+- [Discord](https://charm.sh/chat)
 
 ## License
 
 [MIT](https://github.com/charmbracelet/mods/raw/main/LICENSE)
 
-***
+---
 
 Part of [Charm](https://charm.sh).
 
 <a href="https://charm.sh/"><img alt="The Charm logo" width="400" src="https://stuff.charm.sh/charm-badge.jpg" /></a>
 
+<!--prettier-ignore-->
```

Comment:
What does this do?

Reply:
makes prettier not f up the line bellow :P